enterprise/core-edge/src/main/java/org/neo4j/coreedge/core/EnterpriseCoreEditionModule.java

```diff
@@ -121,6 +126,18 @@ public void registerProcedures( Procedures procedures )
         }
     }
 
+    @Override
+    protected Log authManagerLog( Config config, FileSystemAbstraction fileSystem, JobScheduler jobScheduler )
+            throws IOException
+    {
+        if (securityLog == null)
+        {
+            securityLog = new SecurityLog( config, fileSystem,
+                    jobScheduler.executor( JobScheduler.Groups.internalLogRotation ) );
+        }
```

Comment:
This little piece of lazy creation seem to be duplicated across all enterprise editions. Maybe we could extract a shared helper.